make_html.py

```diff
@@ -277,6 +277,9 @@ def link_sha(sha: Union[port_status_yaml.PortStatusEntry.Source, git.Commit]) ->
 graph = parse_imports(mathlib_dir / 'src')
 graph = nx.transitive_reduction(graph)
 
+# Save topological order to sort by lowest file in import chain.
+topol_order = {node : i  for i, node in enumerate(nx.topological_sort(graph))}
```

Comment:
Instead of sorting in an arbitrary topological order, can you measure the length of the longest path starting at the current node? That way ties can be broken by other columns (such as size of change)

Reply:
Done that. I currently have sorting order `- depth` in there as I'd think the preference is to rather see files with high depth first but I don't really mind.